tests/extend.test.ts

```diff
@@ -0,0 +1,16 @@
+import { GetSecretValueCommand, SecretsManagerClient } from '@aws-sdk/client-secrets-manager';
+
+import '../src/extend.js';
+
+import { mockClient } from 'aws-sdk-client-mock';
+import { describe, expect, it } from 'vitest';
+
+describe('extend', () => {
+  const secretMock = mockClient(SecretsManagerClient);
+
+  it('should make matchers available', () => {
```

Comment:
Not sure if this is just my ruby adoption but I [avoid using should][1] in test descriptions.

```suggestion
  it('makes matchers available', () => {
```

[1]: https://www.betterspecs.org/#should